packages/backend/src/graphql/resolvers/asset.ts

```diff
@@ -192,6 +197,78 @@ export const getFees: AssetResolvers<ApolloContext>['fees'] = async (
   }
 }
 
+export const deleteAsset: MutationResolvers<ApolloContext>['deleteAsset'] =
```

Comment:
I think we should try to keep the resolvers as lean as possible. Maybe, for this particular function (since it's relatively small), we just use `WalletAddress` and `Peer` models directly in the `assetService.delete` function. This would move the business logic to the service instead, and would let you avoid adding new functions to peer and walletAddres services.